package.json

```diff
@@ -15,12 +15,14 @@
     "test": "jest --config=./jest.conf.json",
     "debug": "node --inspect ./node_modules/.bin/jest --watch --config=./jest.conf.json",
     "lint": "./node_modules/.bin/eslint",
-    "precompile": "npm run clean-lib && npm run copy-files && npm run babel",
+    "lint-ts": "./node_modules/.bin/tslint -c tslint.json 'src/components/**/*.ts'",
+    "precompile": "npm run clean-lib && npm run copy-files && npm run typescript && npm run babel",
```

Comment:
Do we still need `npm run typescript` as part of the precompilation process? Should it just be part of the CI build?